daemon/control.c

```diff
@@ -954,6 +985,21 @@ control_handle_message(control_t *control, const ControllerToDaemon *msg, int fd
 		cmld_push_device_cert(control, cert, cert_len);
 	} break;
 
+	case CONTROLLER_TO_DAEMON__COMMAND__SET_PROVISIONED: {
+		cmld_set_device_provisioned();
+		char *provisioned_file =
```

Comment:
remove this file handling here and just do the touch in cmld_set_device_provisioned()
as mentioned above.

Reply:
Done